std/range/interfaces.d

```diff
@@ -208,6 +208,13 @@ interface RandomAccessInfinite(E) : ForwardRange!E {
 interface InputAssignable(E) : InputRange!E {
     ///
     @property void front(E newVal);
+
+    @property E front();
```

Comment:
i'm not sure that's appropriate, the rest of the interfaces only document the functions they add.